Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1559,6 +1563,23 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
             nullptr, // A null string forces the default value.
             readWriteConstructParamFlags);
 
+    /**
+     * WebKitSettings:enable-webrtc:
+     *
+     * Enable or disable support for WebRTC on pages. Enabling this setting implies MediaStream will
+     * be enabled as well.
+     *
+     * See also https://www.w3.org/TR/webrtc/
+     *
+     * Since: 2.38
+     */
+    sObjProperties[PROP_ENABLE_WEBRTC] = g_param_spec_boolean(
+        "enable-webrtc",
+        _("Enable WebRTC"),
+        _("Whether WebRTC content should be handled"),
+        FALSE,
```

Comment:
Should this be `TRUE` by default? Or does it seem better to keep it as `FALSE` for now and change it if/when we feel that the WebRTC implementation is ready for general consumption?

Reply:
For now I think FALSE is better, because the GstWebRTC has too many unimplemented features to be usable. I would re-assess this when we are closer to the 2.38 release.